src/pages/index.js

```diff
@@ -1,21 +1,93 @@
-import React from 'react'
-import { Link } from 'gatsby'
+/* eslint-disable array-bracket-spacing */
+import React, { useState, useEffect } from 'react'
+import { Paper } from '@material-ui/core'
+import Grid from '@material-ui/core/Grid'
+import { useMediaQuery } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import CircularProgress from '@material-ui/core/CircularProgress'
+import DirectorCard from '../components/DirectorCard/DirectorCard'
+import getQueryDataDirectors from '../directors/getQueryDataDirectors'
+import { useTranslation } from 'react-i18next'
 
-import Image from '../components/image'
 import SEO from '../components/seo'
 import App from '../components/app'
 
-const IndexPage = () => (
-  <App>
-    <SEO title="Home" />
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <div style={{ maxWidth: '300px', marginBottom: '1.45rem' }}>
-      <Image />
-    </div>
-    <Link to="/page-2/">Go to page 2</Link>
-  </App>
-)
+import './director.css'
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    textAlign: 'justify',
+    textIndent: 40,
+    margin: 10,
+    padding: 10,
+    '& > * + *': {
+      marginTop: theme.spacing(2)
+    },
+    '& h1': {
+      textAlign: 'center',
+      marginBottom: 50
+    }
+  }
+}))
+const IndexPage = () => {
+  const [loadingIMG, setLoadingIMG] = useState(true)
+  const [directorData, setDirectorData] = useState(null)
+  const { t } = useTranslation()
+  const match = useMediaQuery('(max-width: 945px)')
+  const classes = useStyles()
+  const query = getQueryDataDirectors()
+  useEffect(() => {
+    setDirectorData(query.directors[2 - 1])
```

Comment:
Don't understand ```[2 - 1]```

Reply:
It should be random index for a day. Change for just random index